internal/mempool/mempool.go

```diff
@@ -135,22 +132,17 @@ func NewTxMempool(
 ) *TxMempool {
 
 	txmp := &TxMempool{
-		logger:            logger,
-		config:            cfg,
-		proxyAppConn:      proxyAppConn,
-		height:            -1,
-		cache:             NopTxCache{},
-		duplicateTxsCache: NopTxCacheWithTTL{}, // Default to NOP implementation
-		metrics:           NopMetrics(),
-		txStore:           NewTxStore(),
-		gossipIndex:       clist.New(),
-		priorityIndex:     NewTxPriorityQueue(),
-		heightIndex: NewWrappedTxList(func(wtx1, wtx2 *WrappedTx) bool {
-			return wtx1.height >= wtx2.height
-		}),
-		timestampIndex: NewWrappedTxList(func(wtx1, wtx2 *WrappedTx) bool {
-			return wtx1.timestamp.After(wtx2.timestamp) || wtx1.timestamp.Equal(wtx2.timestamp)
-		}),
+		logger:              logger,
+		config:              cfg,
+		proxyAppConn:        proxyAppConn,
+		height:              -1,
+		cache:               NopTxCache{},
+		duplicateTxsCache:   NopTxCacheWithTTL{}, // Default to NOP implementation
+		metrics:             NopMetrics(),
+		txStore:             NewTxStore(),
+		gossipIndex:         clist.New(),
+		priorityIndex:       NewTxPriorityQueue(),
+		timestampIndex:      NewWrappedTxList(),
```

Comment:
nit: we should probably rename this to something like expirationIndex to represent both height and timestamp?